.circleci/config.yml

```diff
@@ -3,10 +3,9 @@ jobs:
   build:
     working_directory: ~/robocup-training
     docker:
-      - image: silex/emacs:25.1
+      - image: silex/emacs:master-dev
```

Comment:
Actually, it would be safer to choose `26-dev` or `25-dev` instead of `master-dev` since then if Emacs's master branch breaks it won't affect the build here. Always better to use stable releases :P